test/chi/js/dropdown.pug

```diff
@@ -0,0 +1,258 @@
+---
+title: Dropdown
+---
+script(src='/js/chi.js')
+
+h3 Base
+p  Trigger and dropdown enclosed in the same div.
+.chi-dropdown
+  button#dropdown-1.chi-button.chi-dropdown__trigger Dropdown component
+  .chi-dropdown__menu
+    a.chi-dropdown__menu-item(href='#') Item 1
+    a.chi-dropdown__menu-item(href='#') Item 2
+    a.chi-dropdown__menu-item(href='#') Item 3
+script.
```

Comment:
Let's move the scripts to the bottom of the file to have them all consolidated in one place.

Reply:
Moved the scripts to the bottom of the code, please review